docs/user/metal/install_ipi.md

```diff
@@ -177,6 +177,69 @@ pullSecret: ...
 sshKey: ...
 ```
 
+#### Configuration Parameters
```

Comment:
I like this table but we're now documenting many parametersin two places.  Most of these are covered in customization_ipi.md, could we put them there, and maybe only define the absolute minimum required here?

Reply:
I've pushed another patch to move some of the settings as you suggest. Let me know what you think.